### ISSUE TITLE
Serilog Logging not working?

### ISSUE THREAD

**Issue by Domanaik** [KennethScott/SpecOps#8]
According to the Readme Serilog Logging should be used by default and log into the folder /logs
In the precompiled release there is no folder logs. Even if I create it manually, I don't get any logs. 
I tried the following locations: root folder of SpecOps, Scripts and wwwroot

**Comment by KennethScott** [comment]
That's odd - I just downloaded the latest precompiled release, extracted it to a folder setup in IIS, and browsed to it.  The only change I made was to grant myself admin access via the appsettings.json file (AppSettings:SecurityPolicies).  As soon as I browsed to the site, it immediately created the logs folder and started writing to them.  How did you setup the app in IIS exactly?  Is it working as you expected aside from not logging?
